### PR TITLE
feat: new CCC events added

### PR DESCRIPTION
**Pull request by Ivan-Feofanov**

### CODE REVIEW

**Review by BATMAH69 — DISMISSED** [review]

File: l2-bridge-ethereum/src/constants.ts [hunk 1]
```diff
@@ -584,6 +584,54 @@ const BSC_L1_CROSS_CHAIN_CONTROLLER_EVENTS = [
...
+      `${args.oldGuardian} to ${args.newGuardian}`,
+    severity: FindingSeverity.Critical,
+    type: FindingType.Info,
+  },
+  {
+    address: BSC_L1_CROSS_CHAIN_CONTROLLER,
+    event:
+      "event TransactionForwardingAttempted(bytes32 transactionId, bytes32 indexed envelopeId, bytes encodedTransaction, uint256 destinationChainId, address indexed bridgeAdapter, address destinationBridgeAdapter, bool indexed adapterSuccessful, bytes returnData)",
+    alertId: "L1-BRIDGE-MESSAGE-SENT",
+    name: "L1 Cross-chain controller: Message sent",
```
Comment (tamtamchik) [inline l2-bridge-ethereum/src/constants.ts:627]: Add proper icon?

**Review by BATMAH69 — DISMISSED** [review]

File: l2-bridge-ethereum/src/constants.ts (hunk 1 above)
Comment (tamtamchik) [reply l2-bridge-ethereum/src/constants.ts:627]: ℹ️ BNB a.DI: Message sent by the X adapter

**Comment by tamtamchik** [comment]
It looks like we deprecate this bot, and you need to fork it and deploy it under your supervision with all the checks that DAO team needs from this bot

**Review by tamtamchik — CHANGES_REQUESTED** [review]
It looks like we deprecate this bot, and you need to fork it and deploy it under your supervision with all the checks that the DAO team needs from this bot.

**Review by tamtamchik — APPROVED** [review]

**Comment by tamtamchik** [comment]
Disclaimer: approved for easier code transfer to the Dao bot, this code should not be deployed as is.

**Pull request merged**